### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by christophermorin**
Addresses #778.

I had a crack at this. The FAQ questions now follow the template provided in the link within #778. I removed any questions from the current FAQ that were not on the new list. Please let me know if this is what you were aiming for. 

Screenshot 2022-09-19 at 20-14-58 Aqualink

### CODE REVIEW

File: packages/website/src/routes/Faq/index.tsx [hunk 1]
```diff
@@ -88,24 +88,144 @@ const Faq = ({ classes }: FaqProps) => {
...
               </div>
               <div className="row faqq">
                 <div className="col">
                   <Typography className={classes.question} variant="h5">
-                    Who is responsible for the satellite communication costs?
+                    How do I upload surveys?
+                  </Typography>
+                  <p>
+                    Scroll to the bottom of your dashboard for your site and click 
+                    on “ADD NEW SURVEY”. Follow the steps and add comments and
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: I replaced any smart quotes with " ".

File: packages/website/src/routes/Faq/index.tsx [hunk 2]
```diff
@@ -88,24 +88,144 @@ const Faq = ({ classes }: FaqProps) => {
...
+                        <p>
+                          Change the view on the
+                          <a
+                            href="https://aqualink.org/map"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                          >
+                          map
+                          </a> 
+                          to Heat Stress.
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: Fixed. Though I left once instance of Heat Stress, now wrapped in quotes as it is capitalized in the layers menu.

File: packages/website/src/routes/Faq/index.tsx [hunk 3]
```diff
@@ -317,6 +449,13 @@ const Faq = ({ classes }: FaqProps) => {
                   </p>
                 </div>
               </div>
+              <div className="row">
+                <div className="col">
+                  <Typography className={classes.title} variant="h4">
+                    If you have any questions or comments, feel free to contact admin@aqualink.org
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: Not sure if I should have added this one, but the two instances of the email address in the file are now wrapped in anchors, With impromptu subject lines.

File: packages/website/src/routes/Faq/index.tsx [hunk 4]
```diff
@@ -88,24 +88,152 @@ const Faq = ({ classes }: FaqProps) => {
...
+                  </Typography>
+                </div>
+              </div>
+              <div className="row faqq">
+                <div className="col">
+                  <Typography className={classes.question} variant="h5">
+                    How do I apply for an Aqualink Smart Buoy?
+                  </Typography>
+                  <p>
+                    Register the site/s that you are monitoring on Aqualink
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: Fixed

File: packages/website/src/routes/Faq/index.tsx [hunk 5]
```diff
@@ -118,6 +246,18 @@ const Faq = ({ classes }: FaqProps) => {
                   <p>100 meters.</p>
                 </div>
               </div>
+              <div className="row faqq">
+                <div className="col">
+                  <Typography className={classes.question} variant="h5">
+                    Are the Smart Buoys free?
+                  </Typography>
+                  <p>Yes, we cover the costs for the buoys, but you’ll be
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: Fixed, and found one other.

File: packages/website/src/routes/Faq/index.tsx [hunk 6]
```diff
@@ -317,6 +457,20 @@ const Faq = ({ classes }: FaqProps) => {
...
+              <div className="row">
+                <div className="col">
+                  <Typography className={classes.title} variant="h4">
+                  If you have any questions or comments, feel free to contact 
+                  <a
+                      href="mailto:admin@aqualink.org?subject=Questions%20or%20Comments"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      admin@aqualink.org.
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: Fixed.

**Comment by christophermorin** [comment]
I added a screenshot to initial PR comment so it's not buried. 
Admittedly every aspect of your codebase/techstack is a new thing to me so I'm flying by the seat of my pants here figuring out how to make everything work on my end. Hopefully it's in line with what you need.

File: packages/website/src/routes/Faq/index.tsx [hunk 7]
```diff
@@ -88,24 +88,152 @@ const Faq = ({ classes }: FaqProps) => {
...
+                </div>
+              </div>
+              <div className="row faqq">
+                <div className="col">
+                  <Typography className={classes.question} variant="h5">
+                    Which parameters does Aqualink currently support for the dashboard?
+                  </Typography>
+                  <p>Please view
+                  <a
+                      href="https://docs.google.com/document/d/14MfB5iDZqzArZ-MAWuzWnnNs0XsRYK3bchotL-hRMUs/edit"
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: No longer opens in edit.

File: packages/website/src/routes/Faq/index.tsx [hunk 8]
```diff
@@ -88,24 +88,152 @@ const Faq = ({ classes }: FaqProps) => {
...
+              </div>
+              <div className="row faqq">
+                <div className="col">
+                  <Typography className={classes.question} variant="h5">
+                    How many Smart Buoys are available?
+                  </Typography>
+                  <p>
+                    There is no set schedule for when we have Smart Buoys 
+                    available or how big our batch is. Please register your
+                    site/s and reach out to us, and we'll let you know if
```
[the remark this answers is not in the document]
Comment (christophermorin) [reply packages/website/src/routes/Faq/index.tsx]: Fixed.

**Comment by ericboucher** [comment]
This is great @christophermorin thanks! Don't hesitate to ping me if you have questions on the code base or the project in general. You can also email me at eric@ovio.org

While we're at it, could you make `https://onlineshippingcalculator.com/` a clickable link?

**Review by ericboucher — APPROVED** [review]

**Comment by ericboucher** [comment]
And one last edit, on the “Can I see more specifications for the spotter buoy? You can see some of the specifications here.”
You can remove “: More specifications will be released soon” since these are the up-to-date specifications. :)

**Comment by christophermorin** [comment]
@ericboucher I sorted the last two issues. This is a great project and I have more questions than I know what to do with so I'll definitely reach out. Thank you for the opportunity, it was a great learning experience for me.

**Pull request merged**